Updating reading to use db.json

Matt-Eva opened the pull request.

lizbur10 reviewed (changes requested):
Overall, looks great - I just found a couple of things.

The final version of Home.js has this code in it:

const userList = users.map(user => <UserCard key={user.id} user={user}/>);

But UserCard is written to take the individual props:

function UserCard({id, name}) {

One or the other needs to change. The solution code may need to be updated as well, depending on which way you decide to go.

As always, please double check that I'm right and didn't get myself confused :)

Matt-Eva commented:
Nice catch! I just adjusted the UserCard in the reading to use the full user prop. It was already doing so in the actual code file.

The solution was still using `{...user}`, so I adjusted it to use `user={user}`, and made adjustments to UserCard to use the full User prop as well.
